src/app/containers/PageHandlers/withData.jsx

```diff
@@ -2,22 +2,27 @@ import React from 'react';
 import { shape } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
+import nodeLogger from '../../helpers/logger.node';
+
+const logger = nodeLogger(__filename);
 
 const WithData = Component => ({ data }) => {
+  let status;
+
   if (data) {
-    const { data: articleData, status } = data;
-    try {
-      return status === 200 && articleData ? (
-        <Component articleData={articleData} />
-      ) : (
-        <ErrorMain status={status} />
-      );
-    } catch {
-      return <ErrorMain status={status} />;
+    const { data: articleData } = data;
+    let { status } = data;
```

Comment:
[`The let statement declares a block scope local variable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) so its only setting the value inside `if (data) {` and not the one on line 10 :(

Reply:
🙈